docs/BreakingChanges/064 Managed browser hosting controls from the .NET Fra.md

```diff
@@ -0,0 +1,30 @@
+## 64: Managed browser hosting controls from the .NET Framework 1.1 and 2.0
```

Comment:
{
  "Id": "64",
  "Title": "Managed browser hosting controls from the .NET Framework 1.1 and 2.0",
  "Details": "TODO",
  "Suggestion": "TODO",
  "Link": "https://msdn.microsoft.com/en-us/library/hh367887(v=vs.110).aspx#web",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": null,
  "Notes": "These controls are now blocked in IE\nSource analyzer status: Pri 3, Not planned",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis